<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>825</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>825</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="edgeTypeComboBox">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>210</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Directed</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Undirected</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="setVertexLabel">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>80</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="setVertexLabelBtn">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>80</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Set label</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="setWeightEdit">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>250</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>250</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Set weight</string>
    </property>
   </widget>
   <widget class="QLabel" name="edgeType">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>210</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Edge type:</string>
    </property>
   </widget>
   <widget class="QLabel" name="vertexLabel">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>50</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Vertex label:</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>521</width>
      <height>421</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>519</width>
       <height>419</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QGraphicsView" name="graficArea"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>320</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>825</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuSave">
     <property name="title">
      <string>Save</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionImage"/>
     <addaction name="actionObj"/>
    </widget>
    <addaction name="menuSave"/>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionImage">
   <property name="text">
    <string>Image</string>
   </property>
  </action>
  <action name="actionObj">
   <property name="text">
    <string>Obj</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
